### ISSUE TITLE
ASCII punctuation are mangled.

### ISSUE THREAD

**Issue by jrfondren** [katafrakt/stringex-d#3]
This script:

```d
#! /usr/bin/env dub
/+ dub.sdl:
    dependency "stringex" version="~>0.2.0"
+/
enum UNWANTED = "\x1b\x09";

void main() {
    import std.range: iota;
    import std.conv: to;
    import std.stdio: writefln;
    import std.algorithm: filter;
    import stringex.unidecode: unidecode;
    import std.string: indexOf;

    foreach (c; iota(0x00, 0x7F).filter!(c => UNWANTED.indexOf(c) == -1)) {
        auto s1 = to!string(cast(char)c);
        auto s2 = s1.unidecode();
        if (s1 != s2) {
            writefln("preservation failure: %02x (%c) -> %02x (%c)",
                c, cast(char)c, s2[0], s2[0]);
        }
    }
}
```

Has this output:
```
preservation failure: 5b ([) -> 5d (])
preservation failure: 60 (`) -> 27 (')
```

Desired output: none.

My use of this library is actually already disqualified by the ESC and TAB mangling, but the above I argue are obviously just bugs.

**Comment by katafrakt** [comment]
Thank you for the report. This indeed seems to be a bug, I'm able to reproduce it and it does not happen in Ruby stringex, so the problem is somewhere in this lib. I'll take a closer look soon.

**Comment by katafrakt** [comment]
Ok, I released version 0.2.1, where it should be fixed. Can you confirm?

**Comment by jrfondren** [comment]
@katafrakt yep, this version works great.
